extensions/wikia/AdEngine/js/AdEngineRunner.js

```diff
@@ -19,9 +19,11 @@ define('ext.wikia.adEngine.adEngineRunner', [
 	 * @param {function} runAdEngine
 	 */
 	function delayRun(runAdEngine) {
-		var enabledBidders = [],
-			biddersQueue = [],
-			startedByBidders = false;
+		var biddersQueue = [],
+			enabledBidderNames = [],
```

Comment:
I just wanted to prevent keeping two arrays with list of responded bidders (one with names, one with real objects). Especially because they were extended in different places.